src/App.jsx

```diff
@@ -0,0 +1,19 @@
+
+import ColorHarmony from "./components/ColorHarmony/ColorHarmony";
```

Comment:
Should be imported in MainContent instead of App

Reply:
I updated my code but something went wrong this is my old code.

Reply:
I updated please take a look at it.